### PR TITLE
chore(release): bump to 4.6.1

### PR DESCRIPTION
**Pull request by mcous**
## Overview

Bump to 4.6.1 to prepare the hotfix for release

## Review requests

Verify user facing release notes:

- [ ] [api/release-notes.md](https://github.com/Opentrons/opentrons/compare/hotfix_4.6.1...chore_bump-4.6.1?expand=1#diff-15074fc0c51d25eb3b014f9e29d10aca17e85b3414d754d4fe394ebf9804cd88)
- [ ] [app-shell/build/release-notes.md](https://github.com/Opentrons/opentrons/compare/hotfix_4.6.1...chore_bump-4.6.1?expand=1#diff-b11201b87c0040819c75483a892516224ee4af0589c41e72526ec47da2635d0a)

## Risk assessment

N/A

### CODE REVIEW

**Review by amitlissack — APPROVED** [review]

**Review by nusrat813 — APPROVED** [review]
LGTM

File: api/release-notes.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 - Fixed a few issues with the faster protocol analysis method added in the 4.5.0 release.
 - Fixed type annotations of the `ProtocolContext` classes.
 
+## 4.6.1
+
+The 4.6.1 hotfix release contains a small configuration change to to fix an issue with installing the `opentrons` PyPI package on computers running Python 3.8 and later. It does not affect the software running on your OT-2.
```
Comment (bndo) [inline api/release-notes.md]: tiny typo
`change to to fix an`
`change to fix an`

```suggestion
The 4.6.1 hotfix release contains a small configuration change to fix an issue with installing the `opentrons` PyPI package on computers running Python 3.8 and later. It does not affect the software running on your OT-2.
```

File: app-shell/build/release-notes.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 
 ---
 
-# Opentrons App Changes in 4.6.0
+# Opentrons App Changes in 4.6.x
 
-The 4.6.0 release is primarily focused on bug fixes on the robot software side and behind the scenes changes in the app for future work. You shouldn't notice any big changes with this upgrade.
+The 4.6.x release is primarily focused on bug fixes on the robot software side and behind the scenes changes in the app for future work. You shouldn't notice any big changes with this upgrade.
```
Comment (bndo) [inline app-shell/build/release-notes.md]: Should this be `4.6.1`?

File: app-shell/build/release-notes.md [hunk 3]
```diff
@@ -18,6 +18,10 @@ None in the Opentrons App.
 
 None in the Opentrons App.
 
+## 4.6.1
+
+The 4.6.1 hotfix release contains a small configuration change to to fix an issue with installing the `opentrons` PyPI package on computers running Python 3.8 and later. It does not affect the software running in the Opentrons App.
```
Comment (mcous) [inline app-shell/build/release-notes.md]:
```suggestion
The 4.6.1 hotfix release contains a small configuration change to fix an issue with installing the `opentrons` PyPI package on computers running Python 3.8 and later. It does not affect the software running in the Opentrons App.
```

Comment (bndo) [inline app-shell/build/release-notes.md]: `to to` typo

File: app-shell/build/release-notes.md (hunk 2 above)
Comment (mcous) [reply app-shell/build/release-notes.md]: Hm the change from `0` to `x` was intentional, but it does kinda look like someone forgot to fill out a format ("Hello {{user.firstname}}, how are you?")

**Review by mattwelch — APPROVED** [review]
🛳

**Pull request merged**